bundles/org.openhab.binding.doorbird/src/main/java/org/openhab/binding/doorbird/internal/DoorbirdBindingConstants.java

```diff
@@ -0,0 +1,70 @@
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link DoorbirdBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Mark Hilbush - Initial contribution
+ */
+@NonNullByDefault
+public class DoorbirdBindingConstants {
+    public static final String BINDING_ID = "doorbird";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID THING_TYPE_D101 = new ThingTypeUID(BINDING_ID, "d101");
+    public static final ThingTypeUID THING_TYPE_D210X = new ThingTypeUID(BINDING_ID, "d210x");
+
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Stream.of(THING_TYPE_D101, THING_TYPE_D210X)
+            .collect(Collectors.toSet());
+
+    // List of all Channel IDs
+    public static final String CHANNEL_DOORBELL = "doorbell";
+    public static final String CHANNEL_DOORBELL_TIMESTAMP = "doorbellTimestamp";
+    public static final String CHANNEL_DOORBELL_IMAGE = "doorbellImage";
+    public static final String CHANNEL_MOTION = "motion";
+    public static final String CHANNEL_MOTION_TIMESTAMP = "motionTimestamp";
+    public static final String CHANNEL_MOTION_IMAGE = "motionImage";
+    public static final String CHANNEL_LIGHT = "light";
+    public static final String CHANNEL_OPENDOOR1 = "openDoor1";
+    public static final String CHANNEL_OPENDOOR2 = "openDoor2";
+    public static final String CHANNEL_GETIMAGE = "getImage";
+    public static final String CHANNEL_IMAGE = "image";
+    public static final String CHANNEL_IMAGE_TIMESTAMP = "imageTimestamp";
+    public static final String CHANNEL_DOORBELL_HISTORY_INDEX = "doorbellHistoryIndex";
+    public static final String CHANNEL_DOORBELL_HISTORY_IMAGE = "doorbellHistoryImage";
+    public static final String CHANNEL_DOORBELL_HISTORY_TIMESTAMP = "doorbellHistoryTimestamp";
+    public static final String CHANNEL_MOTION_HISTORY_INDEX = "motionHistoryIndex";
+    public static final String CHANNEL_MOTION_HISTORY_IMAGE = "motionHistoryImage";
+    public static final String CHANNEL_MOTION_HISTORY_TIMESTAMP = "motionHistoryTimestamp";
+    public static final String CHANNEL_GET_DOORBELL_MONTAGE = "getDoorbellMontage";
+    public static final String CHANNEL_DOORBELL_IMAGE_MONTAGE = "doorbellMontage";
+    public static final String CHANNEL_GET_MOTION_MONTAGE = "getMotionMontage";
+    public static final String CHANNEL_MOTION_IMAGE_MONTAGE = "motionMontage";
+    public static final String CHANNEL_SIP_HANGUP = "sipHangup";
+    public static final String CHANNEL_RESTART = "restart";
+
+    // List of trigger events
+    public static final String EVENT_PRESSED = "PRESSED";
+    public static final String EVENT_RELEASED = "RELEASED";
+    public static final String EVENT_TRIGGERED = "TRIGGERED";
+    public static final String EVENT_UNTRIGGERED = "UNTRIGGERED";
```

Comment:
It's used in the `startMotionOffJob` method.

Reply:
Sry for my silly question: But what does it mean and what is a use-case for it? I mean an event can be triggered. That is clear. Can it be "untriggered"? Is the event happening over a period of time?

Reply:
> Sry for my silly question

There are no silly questions... :wink:

Most motion sensors with which I'm familiar are in either a triggered (ON) state, or an untriggered (OFF) state. I'm most familiar with zwave motion detectors where they are in an ON state when motion is detected, then after a period of time (sometimes configurable) they change back to an OFF state.

I'm trying to mirror that behavior so that the item referencing the switch profile can be either ON when motion is detected, then become OFF after a period of time. I was using the `UNTRIGGERED` event to cause the transition to OFF after a user-defined number of seconds.

The Doorbird motion sensor is quite sensitive, so it tends to detect motion quite easily (almost to the point of being unusable). Therefore, the specific use case for the Doorbird is give the user the ability to define an interval during which Doorbird motion detection events will not be propagated to the user through the item that references the switch profile.

Reply:
Why don't you use a basic Switch channel for it at all ([system default channel type](https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types) `motion`)?

Reply:
Well... The doorbell and motion channels originally were defined as switches. I later changed them to be triggers, but wanted to preserve the ON/OFF functionality. Maybe I should just change them both back to switches. It will make everything much simpler. WDYT?

Reply:
I am completely okay with `doorbell` channel to be a trigger but imo `motion` should be a Switch.